instrumentation/active_support/lib/opentelemetry/instrumentation/active_support/span_subscriber.rb

```diff
@@ -19,11 +21,13 @@ def self.subscribe(
         tracer,
         pattern,
         notification_payload_transform = nil,
-        disallowed_notification_payload_keys = []
+        disallowed_notification_payload_keys = nil,
+        span_name_formatter = nil
```

Comment:
I wonder if the intent might be more clear if we defaulted this to a passthrough rather than `nil`? Something like:

```ruby
PASSTHROUGH_NAME_FORMATTER = =>(name) { name.dup.freeze }

# …

span_name_formatter = PASSTHROUGH_NAME_FORMATTER
```

Maybe it doesn't matter. Six of one, half a dozen of the other, as they say?

Reply:
I think even though it is symmetric, I am going choose to minimize allocations.